### DISCUSSION TITLE
Implementing Whisper for Summarizing Voice Notes

### DISCUSSION THREAD

**Discussion by edgarsgriga** [openai/whisper#1752] [Q&A] [answered]
Hi everyone,

I'm planning to use OpenAI's Whisper to create summaries from voice notes in bullet points. Should I use the Whisper API or embed the model directly for a personal, small-scale project?

**Key considerations:**

- Feasibility of summarizing transcriptions into bullets
- Whisper API vs. local model for ease and cost
- Recommended model size for English voice notes

I'm not super technical. I have made some small projects and websites, but going to use GPT-4 as my assistant.

Do I have a chance to build this at all? :D 

Thanks for any advice!

**Comment by phineas-pta** [comment, marked as answer]
#1208
#1235
